### PR TITLE
EZEE-1907: Missing notification after publishing/sending for review/publishing in the future

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-1907
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | /no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/config/services/form_processors.yml [hunk 1]
```diff
@@ -11,3 +11,9 @@ services:
         public: true
         tags:
             - { name: kernel.event_subscriber }
+
+    EzSystems\EzPlatformAdminUi\RepositoryForms\Form\Processor\ContentEditNotificationFormProcessor:
+        class: EzSystems\EzPlatformAdminUi\RepositoryForms\Form\Processor\ContentEditNotificationFormProcessor
+        public: true
+        arguments:
+            $siteAccessGroups: '%ezpublish.siteaccess.groups%'
```
Comment (mikadamczyk) [inline src/bundle/Resources/config/services/form_processors.yml]: 👍

File: src/bundle/Resources/config/services/form_processors.yml [hunk 2]
```diff
@@ -11,3 +11,9 @@ services:
         public: true
         tags:
             - { name: kernel.event_subscriber }
+
+    EzSystems\EzPlatformAdminUi\RepositoryForms\Form\Processor\ContentEditNotificationFormProcessor:
+        class: EzSystems\EzPlatformAdminUi\RepositoryForms\Form\Processor\ContentEditNotificationFormProcessor
```
Comment (webhdx) [inline src/bundle/Resources/config/services/form_processors.yml]: No need for `class` property, it's already defined by the name.

File: src/lib/RepositoryForms/Form/Processor/ContentEditNotificationFormProcessor.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+     */
+    public function addPublishMessage(FormActionEvent $event)
+    {
+        if (!$this->isAdminSiteAccess($this->requestStack->getCurrentRequest())) {
+            return;
+        }
+        $this->notificationHandler->success(
+            $this->translator->trans(
+                /** @Desc("Content published.") */
+                'content.published.success'
```
Comment (webhdx) [inline src/lib/RepositoryForms/Form/Processor/ContentEditNotificationFormProcessor.php]: This should've been put in separate translation domain. Right now it's a part of `messages`. I believe we have `content_edit` domain which seems like a better fit.

**Comment by ViniTou** [comment]
@webhdx fixed!

**Pull request merged**